OneSignalSDK/onesignal/core/src/test/java/com/onesignal/core/internal/http/HttpClientTests.kt

```diff
@@ -239,7 +239,7 @@ class HttpClientTests : FunSpec({
 
         mocks.factory.mockResponse = mockSuccessfulResponse
         val response2 =
-            withTimeoutOrNull(999) {
+            withTimeoutOrNull(975) {
```

Comment:
What is significant in changing the number from 975 to 999?

Reply:
Test sometimes failed on "response2 shouldBe null" line below, as `999` was too close to `1000`, which means the background thread could sometimes take priority of the main thread and the main thread could take over 1secound to execute. I ran 200 iterations of the test with 975 and passed.

Reply:
Interesting!